tests/test_model.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from  shortlister import model
+from pathlib import Path
+
+path = "test_role"
+pickle_file_name = "shortlist.pickle"
+
+def test_load_criteria(filepath= "test_role/criteria.csv"):
```

Comment:
Paths should use the file-level variables `path` and `pickle_file_name`, as necessary